indico/modules/search/controllers.py

```diff
@@ -82,6 +88,19 @@ def search_categories(page, q):
         # this would be nicer but then we can't easily paginate...
         return results.total, DetailedCategorySchema(many=True).dump(results.items)
 
+    @staticmethod
+    def search_subcategories(page, q, category_id):
+        # TODO: This must filter subcategories.
+        results = (Category.query
```

Comment:
You can use `category.deep_children_query` to get a query that searches only within children of that category